fabrictestbed_extensions/fablib/network_service.py

```diff
@@ -789,6 +789,25 @@ def get_available_ips(
             logging.warning(f"Failed to get_available_ips: {e}")
             return None
 
+    def get_public_ips(self) -> List[IPv4Address] or List[IPv6Address] or None:
+        """
+        Get list of public IPs assigned to the FabNetv*Ext service
+        :return: List of Public IPs
+        :rtype: List[IPv4Address] or List[IPv6Address] or None
+        """
```

Comment:
It would be nice to have the docstring included in Sphinx generated docs.  I think an entry in docs/source/fablib.rst is needed for that.

Reply:
Added to docs/source/fablib.rst